homeassistant/components/deconz/deconz_device.py

```diff
@@ -130,7 +132,9 @@ def __init__(
         """Set up a scene."""
         super().__init__(device, gateway)
 
-        self._attr_name = device.full_name
+        self.group = self.gateway.api.groups[device.group_id]
+
+        self._attr_name = device.name
```

Comment:
When the name is not set (and thus defaults to `None`), the name of the DeviceInfo is used. Isn't that the same?

Reply:
The Scene still needs its own name for identification I think, which is reflected in the tests which does not need changing